symfony/phpunit-bridge/5.3/bin/phpunit

```diff
@@ -6,9 +6,15 @@ if (!ini_get('date.timezone')) {
 }
 
 if (is_file(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit')) {
-    define('PHPUNIT_COMPOSER_INSTALL', dirname(__DIR__).'/vendor/autoload.php');
-    require PHPUNIT_COMPOSER_INSTALL;
-    PHPUnit\TextUI\Command::main();
+    if (class_exists(PHPUnit\TextUI\Command::class)) {
+        // PHPUnit < 9.x
+        define('PHPUNIT_COMPOSER_INSTALL', dirname(__DIR__).'/vendor/autoload.php');
+        require PHPUNIT_COMPOSER_INSTALL;
+        PHPUnit\TextUI\Command::main();
+    } else {
+        // PHPUnit >= 10.0
+        require dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit';
```

Comment:
the bridge is not compatible with phpunit 10.
I'd suggest removing that part for now at least

Reply:
I'm wondering about a 3rd option :).

Is there a reason why we called `PHPUnit\TextUI\Command::main()` for PHPUnit 9.5 vs requiring the executable? Could this new code be used in ALL cases?

> the bridge is not compatible with phpunit 10.

I understand this... but since you CAN easily install `phpunit-bridge` and PHPUnit 10, if there is a straightforward way to get this file working today, why wait until tomorrow to do it? :)

Reply:
> the bridge is not compatible with phpunit 10.

I would argue that at all times the `phpunit-bridge` should preventively declare a conflict with future major versions of PHPunit until tested/adapted, it already declares some:
```
    "conflict": {
        "phpunit/phpunit": "<7.5|9.1.2"
    },
```
I'd suggest adding `|>=11` once v10 support is implemented (won't make any difference now as Composer will just install the older version not declaring the conflict).

Right now running `symfony new test` followed by `composer req symfony/phpunit-bridge phpunit/phpunit` on a new project will create a broken project.